README.md

```diff
@@ -1,5 +1,4 @@
 # Mathician
 <img height="200" width="200" src="https://cdn.discordapp.com/attachments/980115259534749746/984839027398160394/Untitled.png"/>
-Mathician is a math package that uses the math package (some of it) and it's own code. Created by Arjun Sharda with the video https://www.youtube.com/watch?v=zhpI6Yhz9_4, it is one simple python package that can be used, although very basic and not many features.
 
-Mathician uses the ``math`` package aswell in it's code, and it's own code. Mathician is open-sourced and is free for everyone to use under the MIT license.
+Mathician is a math package that uses the math package (some of it) and it's own code. It is created by Arjun Sharda with the video https://www.youtube.com/watch?v=zhpI6Yhz9_4, and it is one simple python package that can be used, although very basic and does not have many features. Mathician is open-sourced and is free for everyone to use under the MIT license.
```

Comment:
Mathician is a math package that uses the math package (some of it) and it's own code. It is created by Arjun Sharda with the video https://www.youtube.com/watch?v=zhpI6Yhz9_4, and it is one simple python package that can be used, although very basic and does not have many features. Mathician is open-sourced and is free for everyone to use, clone, etc under the MIT license.